docs/topics/mta-web-controlling-task-order-with-task-manager.adoc

```diff
@@ -0,0 +1,22 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-08-07
+:_mod-docs-content-type: PROCEDURE
+
+[id="mta-web-controlling-task-order-with-task-manager_{context}"]
+= Controlling the order of task execution
+
+You can use *Task Manager* to preempt a {ProductFullName} task you have scheduled for execution. 
+
+[NOTE]
+====
+You can only set *Preemption* for automated tasks (language discovery or technology discovery).
+====
```

Comment:
You can enable preemption on any task.
However, it's only has an affect on higher priority tasks such as analysis.  Thus, only useful to enable on analysis tasks.

Reply:
@mguetta1, please, check the new text we created together with @jortel.

Reply:
The 2nd.
> lower-priority tasks are candidates to be preempted means that they are candidates to be rescheduled in favor of higher tasks?